_posts/2025-01-31-black-history-month.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "US-RSE Celebrates Black History Month 2025"
+tags: [dei, black-history]
+author: Lance Parsons, Cordero Core
+date: 2025-01-31
+---
+
+![Black History Month Logo]({{ site.baseurl }}/assets/img/BHMlogo.png){:
+.align-center .width-half}
+
+US-RSE’s [DEI working group (DEI-WG)](https://us-rse.org/wg/dei/) is proud to
+lead the US-RSE’s participation in celebrating [Black History
+Month](https://www.blackhistorymonth.gov/).
+
+First up, we’ll be sharing stories throughout the month that recognize and
+celebrate African Americans who have inspired our members through their
+accomplishments in their careers and their personal stories. We’ll be
+publishing these posts throughout February to highlight such people, where we
+talk about the person and tie their work and life to the RSE movement.
+
+* A. Phillip Randolph \-
+  [Wikipedia](https://en.wikipedia.org/wiki/A._Philip_Randolph)
+* Check back here for links as articles are posted
+
+In addition, we’re excited to host a **Zoom discussion on Thursday, February
+20th at noon PT / 3 PM ET**. [@Cordero
+Core](https://usrse.slack.com/team/U0523JRUDE3) shared one of his favorite
+YouTube channels with us from Garrison Hayes. We invite you to watch the three
+episodes linked below from [Garrison Hayes' YouTube
+channel](https://www.youtube.com/@GarrisonHayes/featured), and then come
+prepared for a lively, engaging, and thought provoking conversation with us
+over Zoom on Thursday, February 20th and continuing in the [`#dei-discussion`
+channel](https://usrse.slack.com/archives/C01C8CJQ7AP) on Slack (details and
+link in the channel). For those that aren't familiar, Garrison explores a
+variety of interesting questions, usually at the intersection of Black history,
+politics and current events.
+
+* [Why the right is so obsessed with
+  DEI](https://www.youtube.com/watch?v=ARKvxeNCp84)
+* [Tim Scott and the "token black
+  guy"](https://www.youtube.com/watch?v=TcEOpjV1Upc)
+* [Why are Black people still
+  Christian?](https://www.youtube.com/watch?v=lKSF1huXOuw)
+
+Finally, we wanted to share with you some articles, shows, and resources we
+found engaging and we invite you to share your own, either on Slack or with the
+hashtag `#usrse-blackhistory`.
+
+### **Events/Conferences**
+
+* [**US-RSE Zoom Discussion**](https://usrse.slack.com/archives/C01C8CJQ7AP) -
+  Thursday, February 20 - Garrison Hayes episodes on DEI, the "token black
+  guy", and Black Christians - Register on Slack in the [`#dei-discussion`
+  channel](https://usrse.slack.com/archives/C01C8CJQ7AP)
+* [Black Tech Week](https://blacktechweek.com/) \- July 14-16 at the Aronoff
+  Center in Cincinnati, OH
+* [BitCon 2026](https://bitcon.blacksintechnology.net/) \- BITCON is the annual
```

Comment:
```suggestion
* [BITCON 2025](https://bitcon.blacksintechnology.net/) \- BITCON is the annual
```
Just since I think BIT is an acronym for Blacks In Technology. So, maybe it's best to make it all caps. Although, their website has the title "Bitcon 2025" (in web search). Also, realized we had 2026 for the date 😝